tests/upgrade/postgres_run.sh

```diff
@@ -10,7 +10,7 @@ TEST_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
 EARLIER_TAG="4.1.3"
 EARLIER_SHA="8a4677e3d45ebc4f065ec052d1d66d6ead2084bb"
 CURRENT_TAG="$(make --quiet --no-print-directory tag)"
-PREVIOUS_RELEASES=("4.1.3" "4.2.5" "4.3.8" "4.4.6" "4.5.5")
+PREVIOUS_RELEASES=("4.1.3" "4.2.5" "4.3.8" "4.4.6" "4.5.5" "4.6.0")
```

Comment:
```suggestion
PREVIOUS_RELEASES=("4.1.3" "4.2.5" "4.3.8" "4.4.7" "4.5.5" "4.6.0")
```

@SimonBaeumer 
IIUC this is the list of last z-releases per supported stream.
We walk 
* from the `EARLIER_TAG` set at https://github.com/stackrox/stackrox/blob/master/tests/upgrade/postgres_run.sh#L10
* through the latest version for each y-stream https://github.com/stackrox/stackrox/blob/master/tests/upgrade/postgres_run.sh#L169-L179
* and finally upgrade to the current tag under test https://github.com/stackrox/stackrox/blob/master/tests/upgrade/postgres_run.sh#L181-L186

to capture migration issues between versions.  

@dashrews78 
Can we remove the 4.1, 4.2, 4.3 streams and update the `EARLIER_TAG` to `4.4.7`?
https://access.redhat.com/support/policy/updates/rhacs

Reply:
Sadly no.  It is on my list to do in 4.8 though I will take a look and re-evaluate if I can do it in 4.7.  As long as the code supports an upgrade from 4.1 to present we should keep the test.  I was going to remove that for 4.6 but discovered a bug.  
https://issues.redhat.com/browse/ROX-26374